### DISCUSSION TITLE
Question about implementing updates to Weblate

### DISCUSSION THREAD

**Discussion by Panquesito7** [WeblateOrg/weblate#5711] [Q&A] [answered]
I am working on an Open Source Weblate project (called [TheAlgorithms](https://hosted.weblate.org/projects/TheAlgorithms/)). When trying to ask for approval for an Open Source project, it asks us to implement [this](https://docs.weblate.org/es/latest/admin/continuous.html#updating-repositories), so it can be approved. But, I'm kinda confused there. It doesn't explain that well how to do it.

Is there an easier way for how to implement it? Or a simplified version?
Thanks in advance! 🙂

**Comment by nijel** [comment, marked as answer]
It's linked from there - https://docs.weblate.org/es/latest/admin/continuous.html#github-setup

**Reply by nijel** [reply]
I've made that more visible in 3225cc558b6aa03d69fdab863fd8c9ad59c1ac0f.
